### PR TITLE
en: add daily check

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
### What is changed, added or deleted? (Required)

add daily check

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-dm/pull/91

### CODE REVIEW

**Comment by WangXiangUSTC** [comment]
@GMHDBJD PTAL

File: en/daily-check.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+This document summarizes the daily inspection methods of the TiDB Data Migration (DM):
+
++ Method 1: Execute the `query-status` command to view the task running status and related error output. See [Query Status](query-status.md) for details.
+
++ Method 2: If Prometheus and Grafana are correctly deployed when using DM-Ansible to deploy the DM cluster, then you can veiw DM's dashboard in Grafana. For example, Grafana's address is `172.16.10.71`, you can open <http://172.16.10.71:3000> in your browser to enter the Grafana, and then choose DM's Dashboard to view DM related monitoring items. Refer to [Data Migration Monitoring Metrics](monitor-a-dm-cluster.md) for specific monitoring items.
+
++ Method 3: Check the DM running status and related errors through the log file.
+
+    - DM-master's log directory: set by the DM-master's parameter `--log-file`. If DM-Ansible is used to deploy DM, the log directory is located in `{ansible deploy}/log/dm-master.log` of DM-master node.
+    - DM-worker's log directory: set by DM-worker's parameter `--log-file`. If DM-Ansible is used to deploy DM, the log directory is located in `{ansible deploy}/log/dm-worker.log` of the DM-worker node.
```
Comment (toutdesuite) [inline en/daily-check.md:18]:
```suggestion
    - DM-worker log directory: It is specified by the `--log-file` DM-worker process parameter. If DM is deployed using DM-Ansible, the log directory is `{ansible deploy}/log/dm-worker.log` in the DM-worker node.
```

**Comment by toutdesuite** [comment]
@WangXiangUSTC Please update TOC, thanks

**Comment by WangXiangUSTC** [comment]
> @WangXiangUSTC Please update TOC, thanks

I updated it, but the TOC is different with the Chinese version, we need to refactor it later.

**Comment by toutdesuite** [comment]
> > @WangXiangUSTC Please update TOC, thanks
> 
> I updated it, but the TOC is different with the Chinese version, we need to refactor it later.

ok~

**Pull request merged**